lib/Crypto/cryptomanager.h

```diff
@@ -313,6 +313,19 @@ class SAILFISH_CRYPTO_API CryptoManager : public QObject
     Q_DECLARE_FLAGS(Operations, Operation)
     Q_FLAG(Operations)
 
+    enum VerificationResult {
+        VerificationSucceeded,
+        VerificationFailed,
+        VerificationInvalidSignature,
+        VerificationValidSignatureExpired,
+        VerificationValidSignatureWithExpiredKey,
+        VerificationValidSignatureWithRevokedKey,
+        VerificationInvalidKey,
+
+        VerificationResultUnknown = 255
```

Comment:
For all other enums, Unknown had value 0.  I guess it's not important, but is there any reason why a different value was chosen for this one?

Reply:
I wrote it thinking of Qt APIs which generally have NoError/Success = 0. Though, a lot of them have unknown/generic-type error = 1, rather than = 255. The thing is that VerificationResultUnknown doubles as a busy / result-unknown value -- not sure if we want that, or whether it should always be the generic "VerificationFailed" even in the busy case.